File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.ts
function reverseWords(s: string): string {
    let words=s.split(" ");
   let result=[];
   for(let i=0;i<words.length;i++){
    let revWord=words[i].split("").reverse().join("");
    result.push(revWord);
   }
   return result.join(" ");
};

File: 0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.cs
public class Solution {
    public string ReverseWords(string s) {
       List<string> result=new List<string>();
       string[] words=s.Split(' ');
       for(int i=0;i<words.Length;i++){
        string RevWord=new string(words[i].Reverse().ToArray());
        result.Add(RevWord);
       }
       return string.Join(" ",result);
    }
}